src/main/steps/applicant/yourdocuments/alldocuments/allDocumentsGetController.ts

```diff
@@ -44,7 +46,9 @@ export default class AllDocumentsGetController {
       if (error) {
         throw error;
       }
-      res.redirect(url + getViewDocumentUrl(req.params.docType));
+      req.params.docType === DocType.RESPONSE_TO_CA || req.params.docType === DocType.AOH_TO_CA
```

Comment:
Add brackets for || condition

Reply:
The linter throws an error when I add brackets here. I've checked and the condition works for different docType values